### PR TITLE
Case sensitive support for label mapping

### PR DESCRIPTION
**Pull request by lpichler**
related to https://github.com/ManageIQ/manageiq/pull/20289  or (https://github.com/ManageIQ/manageiq/pull/20289#discussion_r504746433)
Described in second point [here](https://github.com/ManageIQ/manageiq-ui-classic/pull/7273#issuecomment-698995573)

@miq-bot assign @agrare

### CODE REVIEW

File: app/models/manageiq/providers/azure/cloud_manager.rb [hunk 1]
```diff
@@ -45,6 +45,10 @@ def insights?
     end
   end
 
+  def case_insensitive_labels?
+    true
```
Comment (agrare) [inline app/models/manageiq/providers/azure/cloud_manager.rb:49]: This should be `case_sensitive_labels? false`

**Comment by chessbyte** [comment]
shouldn't the overridden methods here be `case_sensitive_labels?`

**Review by agrare — CHANGES_REQUESTED** [review]

File: app/models/manageiq/providers/azure/cloud_manager.rb (hunk 1 above)
Comment (lpichler) [reply app/models/manageiq/providers/azure/cloud_manager.rb]: yes, it was leftover and I removed it now.

we have such method on azure's  persister

thanks

File: app/models/manageiq/providers/azure/inventory/persister.rb [hunk 2]
```diff
@@ -23,4 +23,10 @@ def initialize(manager, target = nil, collector = nil)
 
     initialize_inventory_collections
   end
+
+  private
+
+  def case_sensitive_labels?
+    true
```
Comment (agrare) [inline app/models/manageiq/providers/azure/inventory/persister.rb:30]: This should be false right?

**Comment by lpichler** [comment]
> shouldn't the overridden methods here be `case_sensitive_labels?`

yes, it is fixed now 👍 thanks

**Review by agrare — APPROVED** [review]

File: app/models/manageiq/providers/azure/inventory/persister.rb (hunk 2 above)
Comment (lpichler) [reply app/models/manageiq/providers/azure/inventory/persister.rb]: yea good eye, it is fixed.

**Pull request merged**